src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2165,6 +2165,17 @@ bool AdaptDispatch::EnableAlternateScroll(const bool enabled)
     return success;
 }
 
+//Routine Description:
+// Enable "bracketed paste mode".
+//Arguments:
+// - enabled - true to enable, false to disable.
+// Return value:
+// True if handled successfully. False otherwise.
+bool AdaptDispatch::EnableBracketedPasteMode(const bool /*enabled*/) noexcept
+{
+    return NoOp();
```

Comment:
If I'm not mistaken, this is actually the only usage of `NoOp`. Makes me wonder should I use `NoOp` or just return false?

Reply:
Wow, cool! Let's use NoOp for now.